Nur in sysinfo-0.1.0a: libvdr-sysinfo.so.
Nur in sysinfo-0.1.0a: libvdr-sysinfo.so.1.4.1.
Nur in sysinfo-0.1.0a: sysinfo.o.
diff -ru sysinfo-0.1.0a-orig/sysinfoosd.h sysinfo-0.1.0a/sysinfoosd.h
--- sysinfo-0.1.0a-orig/sysinfoosd.h	2006-07-08 21:25:20.317015369 +0200
+++ sysinfo-0.1.0a/sysinfoosd.h	2006-07-08 21:25:36.534592954 +0200
@@ -40,8 +40,8 @@
 	
 	void SendPalette(void);
 	void SetColor(int Index, int Red, int Green, int Blue, int Alpha);
-	void cSysInfoOsd::PleaseWait();
-	void cSysInfoOsd::ShowData();
+	void PleaseWait();
+	void ShowData();
 	
 	void DrawMenu();
 	void DisplayBitmap(); 
Nur in sysinfo-0.1.0a: sysinfoosd.o.
